### PR TITLE
Rapayment length to update when typing new value

### PR DESCRIPTION
**Pull request by RosDiaf**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6695

### Change description ###

**Bugs:**

1. After entering repayment plan and re-visit the same page, '**Length of repayment plan**' label is empty. **(Fixed)**

2. Length of repayment plan update the value when selecting radio box frequency only, not when typing different amount value for payment plan too. **(Fixed)**
 
211544034-dd868f87-9a34-4790-b665-e31913c201d6

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by RosDiaf** [comment]
**Bugs:**

1. After entering repayment plan and re-visit the same page, '**Length of repayment plan**' label is empty. **(Fixed)**

2. Length of repayment plan update the value when selecting radio box frequency only, not when typing different amount value for payment plan too. **(Fixed)**
 
bug

**Review by KyleMulvihillV1 — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

File: src/main/assets/js/calculate-length-repayment.js [hunk 1]
```diff
@@ -45,12 +45,22 @@
   hideAll();
 
   if (repaymentFrequency) {
+
+    window.addEventListener('load', () => {
+      repaymentFrequencyValue = repaymentFrequency.querySelector(repaymentFrequencyRadioBox);
+      if (repaymentFrequencyValue && repaymentFrequencyValue.value) {
```
Comment (TatsyMaslova) [inline src/main/assets/js/calculate-length-repayment.js:51]: repaymentFrequencyValue?.value probably would be better especially if you are repeating the code elsewhere

Comment (RosDiaf) [reply src/main/assets/js/calculate-length-repayment.js]: Added

**Review by portizdev — APPROVED** [review]

File: src/main/assets/js/calculate-length-repayment.js [hunk 2]
```diff
@@ -45,12 +45,22 @@
   hideAll();
 
   if (repaymentFrequency) {
+
+    window.addEventListener('load', () => {
+      repaymentFrequencyValue = repaymentFrequency.querySelector(repaymentFrequencyRadioBox);
+      if (repaymentFrequencyValue && repaymentFrequencyValue?.value) {
```
Comment (TatsyMaslova) [inline src/main/assets/js/calculate-length-repayment.js:51]: still not happy sorry. you don't need the first part. the first part is checked by the "?" so ` repaymentFrequencyValue?.value` is the same as` repaymentFrequencyValue && repaymentFrequencyValue.value` so you don't need the first part of the && if you are using ?. hope it makes sense

Comment (RosDiaf) [reply src/main/assets/js/calculate-length-repayment.js]: changes added

**Review by TatsyMaslova — APPROVED** [review]

**Pull request merged**